editor/src/components/canvas/canvas-strategies/absolute-reparent-strategy.tsx

```diff
@@ -83,12 +83,16 @@ export const absoluteReparentStrategy: CanvasStrategy = {
     )
     const newParent = reparentResult.newParent
     const moveCommands = absoluteMoveStrategy.apply(canvasState, interactionState, strategyState)
-    const providesBoundsForAbsoluteChildren = MetadataUtils.findElementByElementPath(
-      strategyState.startingMetadata,
-      newParent,
-    )?.specialSizeMeasurements.providesBoundsForAbsoluteChildren
+    const providesBoundsForAbsoluteChildren =
+      MetadataUtils.findElementByElementPath(strategyState.startingMetadata, newParent)
+        ?.specialSizeMeasurements.providesBoundsForAbsoluteChildren ?? false
+    const parentIsStoryboard = newParent == null ? false : EP.isStoryboardPath(newParent)
```

Comment:
can this "point" to the canvas even if the mouse is over a (non-eligible to be parent) element? or only if the mouse is over the grey background of the canvas?

Reply:
Currently only if the mouse is over the canvas itself.